packages/css/fieldset.css

```
@@ -1,41 +1,41 @@
 @layer fds.fieldset {
-  .fieldset {
+  .fds-fieldset {
     margin: 0;
     padding: 0;
     border: 0;
     min-width: 0;
   }
 
-  .withSpacing {
+  .fds-fieldset--spacing {
     display: flex;
     flex-direction: column;
     gap: var(--fds-spacing-2);
   }
 
-  .description {
+  .fds-fieldset__description {
     color: var(--fds-semantic-text-neutral-subtle);
     font-weight: 400;
   }
 
-  .legend {
+  .fds-fieldset__legend {
     display: contents;
   }
 
-  .readonly .legendContent {
+  .fds-fieldset--readonly .fds-fieldset__legend__content {
     display: inline-flex;
   }
 
-  .readonly .padlock {
+  .fds-fieldset--readonly .fds-fieldset__padlock {
```

I have used this element name for the same "readonly icon" elsewhere. 

Maybe it would be better called `readonly-icon` 🤔 

Suggest we keep it the same so we can search and replace later if we decide to change it.

```suggestion
  .fds-fieldset--readonly .fds-fieldset__readonly__icon {
```